addons/implied-repos/common/src/main/java/org/commonjava/indy/implrepo/change/ImpliedRepositoryDetector.java

```diff
@@ -399,19 +400,29 @@ else if ( !config.isIncludeSnapshotRepos() && !repo.isReleasesEnabled() )
                             e );
                 }
 
+                final RemoteRepository ref = creator.createFrom( gav, repo, LoggerFactory.getLogger( creator.getClass() ) );
+                if ( ref == null )
+                {
+                    logger.warn(
+                            "ImpliedRepositoryCreator didn't create anything for repo: {}, specified in: {}. Skipping.",
+                            repo.getId(), gav );
+                    continue;
+                }
+
+                if ( rrs != null && !rrs.isEmpty() )
+                {
+                    rrs = rrs.stream()
+                             .filter( rr -> rr.isAllowReleases() == repo.isReleasesEnabled() )
```

Comment:
Wouldn't it be better to compare with the ref instance? It should match, but who knows... :-)